resources/js/components/utils/InitializeTanstackQuery.tsx

```diff
@@ -90,20 +90,12 @@ export const InitializeTanstackQuery: ParentComponent = (props) => {
 };
 
 /**
- * Initialize some of required queries beforehand
+ * Initialize some of required queries beforehand.
  */
 const Content: ParentComponent = (props) => {
   const facilitiesQuery = createQuery(System.facilitiesQueryOptions);
-
   return (
-    <QueryBarrier
-      queries={[facilitiesQuery]}
-      Pending={() => (
-        <div class="h-screen flex justify-center items-center">
-          <MemoLoader size={300} />
-        </div>
-      )}
-    >
+    <QueryBarrier queries={[facilitiesQuery]} Pending={MemoLoader}>
```

Comment:
To potrzebne? Wydaje się nadgorliwe z tym facilitiesQuery. Może niech sobie ściągają te elementy co faktycznie potrzebują, np. routes albo obecnie AccessBarier.

Reply:
To również zmieniam w PR o przyspieszeniu ładowania :) Wolałbym już tego teraz nie łączyć.

Reply:
Ok. Tam też jest nadgorliwie, możemy tam podyskutować.